docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -0,0 +1,292 @@
+Remote Attestation Collaterals
+=====
+
+Adding attestation APIs to support collaterals for remote attestation.  Collaterals are information from a second source that the user/challenger can provide for attestation verification.
```

Comment:
FYI, the IETF RATS WG is now using the term "Endorsements" for things like PCK cert chain, CRL, etc.
Endorsements might be provided to a verifier by an Attester (along with the Evidence), or might be obtained separately by a verifier.   What's the relationship between Collaterals and Endorsements?

If they're the same, then I'd recommend changing to use IETF terminology.

Reply:
Will change to use IETF terminology.  Thank you Dave.  Just note that to get the evidence we will still be using oe_get_report().